suite-native/module-settings/src/components/FAQInfoPanel.tsx

```diff
@@ -17,6 +20,250 @@ const AccordionContentText = ({
+                            </BulletListItem>
+                        </Box>
+
+                        <AccordionContentText
+                            translationKey="moduleSettings.faq.bluetoothEnabled.android.3.answer.footer"
+                            values={{
+                                link: chunk => (
+                                    <Link
+                                        label={chunk}
+                                        textVariant="label"
+                                        href={`${TREZOR_SUPPORT_DEVICE_URL}#open-chat`}
+                                        isUnderlined
+                                    />
+                                ),
+                            }}
+                        />
+                    </Box>
+                ),
+            },
+            {
+                title: <Translation id="moduleSettings.faq.usbEnabled.4.question" />,
+                content: (
+                    <Box style={{ position: 'relative' }}>
+                        <BulletListItem variant="label">
+                            <Translation id="moduleSettings.faq.usbEnabled.4.answer.0" />
+                        </BulletListItem>
+                        <BulletListItem variant="label">
+                            <Translation id="moduleSettings.faq.usbEnabled.4.answer.1" />
+                        </BulletListItem>
+                        <BulletListItem variant="label">
+                            <Translation id="moduleSettings.faq.usbEnabled.4.answer.2" />
+                        </BulletListItem>
+                        <BulletListItem variant="label">
+                            <Translation id="moduleSettings.faq.usbEnabled.4.answer.3" />
+                        </BulletListItem>
+                        <BulletListItem variant="label">
+                            <Translation id="moduleSettings.faq.bluetoothEnabled.android.4.answer.4" />
+                        </BulletListItem>
+                    </Box>
+                ),
+            },
+            {
+                title: <Translation id="moduleSettings.faq.usbEnabled.5.question" />,
+                content: (
+                    <AccordionContentText translationKey="moduleSettings.faq.usbEnabled.5.answer" />
+                ),
+            },
+            {
+                title: <Translation id="moduleSettings.faq.usbEnabled.6.question" />,
+                content: (
+                    <AccordionContentText translationKey="moduleSettings.faq.usbEnabled.6.answer" />
+                ),
+            },
+            {
+                title: <Translation id="moduleSettings.faq.usbEnabled.7.question" />,
+                content: (
+                    <AccordionContentText translationKey="moduleSettings.faq.usbEnabled.7.answer" />
+                ),
+            },
+        ]}
+    />
+);
+
+const BluetoothIOSFAQ = ({ coinLabel }: { coinLabel: string }) => (
```

Comment:
This is a great example why it's better to use CamelCase for shortcuts:
```suggestion
const BluetoothIosFaq = ({ coinLabel }: { coinLabel: string }) => (
```